opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -90,21 +90,24 @@
 def _import_config_components(
     selected_components: List[str], entry_point_name: str
 ) -> Sequence[Tuple[str, object]]:
-    component_entry_points = {
-        ep.name: ep for ep in iter_entry_points(entry_point_name)
-    }
-    component_impls = []
+
+    component_implementations = []
+
     for selected_component in selected_components:
-        entry_point = component_entry_points.get(selected_component, None)
-        if not entry_point:
-            raise RuntimeError(
-                f"Requested component '{selected_component}' not found in entry points for '{entry_point_name}'"
```

Comment:
Done we raise this descriptive error anymore?

Reply:
Added the descriptive error again.